### ISSUE TITLE
Unable to install 1.31.0 from PyPI on Ubuntu 15.10 using buildout

### ISSUE THREAD

**Issue by emorozov** [core-api/python-client#85]
I'm receiving the following error:
```
Getting distribution for 'coreapi==1.31.0'.
zip_safe flag not set; analyzing archive contents...
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "build/bdist.linux-x86_64/egg/setuptools/command/easy_install.py", line 2291, in main
    
  File "/usr/lib/python2.7/distutils/core.py", line 151, in setup
    dist.run_commands()
  File "/usr/lib/python2.7/distutils/dist.py", line 953, in run_commands
    self.run_command(cmd)
  File "/usr/lib/python2.7/distutils/dist.py", line 972, in run_command
    cmd_obj.run()
  File "build/bdist.linux-x86_64/egg/setuptools/command/easy_install.py", line 409, in run
    
  File "build/bdist.linux-x86_64/egg/setuptools/command/easy_install.py", line 645, in easy_install
    
  File "build/bdist.linux-x86_64/egg/setuptools/command/easy_install.py", line 694, in install_item
    
  File "build/bdist.linux-x86_64/egg/setuptools/command/easy_install.py", line 875, in install_eggs
    
  File "build/bdist.linux-x86_64/egg/setuptools/command/easy_install.py", line 1114, in build_and_install
    
  File "build/bdist.linux-x86_64/egg/setuptools/command/easy_install.py", line 1100, in run_setup
    
  File "build/bdist.linux-x86_64/egg/setuptools/sandbox.py", line 246, in run_setup
  File "/usr/lib/python2.7/contextlib.py", line 35, in __exit__
    self.gen.throw(type, value, traceback)
  File "build/bdist.linux-x86_64/egg/setuptools/sandbox.py", line 195, in setup_context
  File "/usr/lib/python2.7/contextlib.py", line 35, in __exit__
    self.gen.throw(type, value, traceback)
  File "build/bdist.linux-x86_64/egg/setuptools/sandbox.py", line 166, in save_modules
  File "build/bdist.linux-x86_64/egg/setuptools/sandbox.py", line 141, in resume
  File "build/bdist.linux-x86_64/egg/setuptools/sandbox.py", line 154, in save_modules
  File "build/bdist.linux-x86_64/egg/setuptools/sandbox.py", line 195, in setup_context
  File "build/bdist.linux-x86_64/egg/setuptools/sandbox.py", line 243, in run_setup
  File "build/bdist.linux-x86_64/egg/setuptools/sandbox.py", line 273, in run
  File "build/bdist.linux-x86_64/egg/setuptools/sandbox.py", line 242, in runner
  File "build/bdist.linux-x86_64/egg/setuptools/sandbox.py", line 46, in _execfile
  File "/tmp/easy_install-SPbkDb/coreapi-1.31.0/setup.py", line 83, in <module>
    
  File "/usr/lib/python2.7/distutils/core.py", line 151, in setup
    dist.run_commands()
  File "/usr/lib/python2.7/distutils/dist.py", line 953, in run_commands
    self.run_command(cmd)
  File "/usr/lib/python2.7/distutils/dist.py", line 972, in run_command
    cmd_obj.run()
  File "build/bdist.linux-x86_64/egg/setuptools/command/bdist_egg.py", line 209, in run
  File "build/bdist.linux-x86_64/egg/setuptools/command/bdist_egg.py", line 245, in zip_safe
  File "build/bdist.linux-x86_64/egg/setuptools/command/bdist_egg.py", line 355, in analyze_egg
  File "build/bdist.linux-x86_64/egg/setuptools/command/bdist_egg.py", line 392, in scan_module
ValueError: bad marshal data (unknown type code)
An error occurred when trying to install coreapi 1.31.0. Look above this message for any errors that were output by easy_install.
While:
  Installing django.
  Getting distribution for 'coreapi==1.31.0'.
Error: Couldn't install: coreapi 1.31.0
```

I guess, there're python files in the distribution pre-compiled for the wrong version of the python interpreter.

**Comment by tomchristie** [comment]
Thanks for the report! :

**Comment by tomchristie** [comment]
Should now be resolved, with `1.31.1` now on PyPI.
